spec/rubocop/cop/rspec/named_subject_spec.rb

```diff
@@ -60,6 +60,96 @@ def foo
     end
   end
 
+  context 'when EnforcedStyle is :named_only' do
+    let(:cop_config) { { 'EnforcedStyle' => :named_only } }
+
+    it 'ignores subject where declaration is not named' do
+      expect_no_offenses(<<-RUBY)
+        RSpec.describe User do
+          subject { described_class.new }
+
+          it "is a User" do
+            expect(subject).to be_a(User)
+          end
+
+          it "is valid" do
+            expect(subject.valid?).to be(true)
+          end
+
+          it "is new" do
+            expect(subject).to be_new_record
+          end
+        end
+      RUBY
+    end
+
+    it 'ignores subject! where declaration is not named' do
+      expect_no_offenses(<<-RUBY)
+        RSpec.describe User do
+          subject! { described_class.new }
+
+          it "is a User" do
+            expect(subject).to be_a(User)
+          end
+
+          it "is valid" do
+            expect(subject.valid?).to be(true)
+          end
+
+          it "is new" do
+            expect(subject).to be_new_record
+          end
```

Comment:
This is redundant. I see other examples in this spec have the same duplication. I suggest to remove and only keep:
```ruby
         it "is valid" do
            expect(subject.valid?).to be(true)
          end
```